agr_literature_service/lit_processing/pdf2tei/pdf2tei.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+from io import BytesIO
+
+import requests
+from fastapi import UploadFile
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from agr_literature_service.api.crud.referencefile_crud import get_main_pdf_referencefile_id, download_file, file_upload
+from agr_literature_service.api.crud.workflow_tag_crud import get_ref_ids_with_workflow_status, \
+    transition_to_workflow_status
+from agr_literature_service.api.database.config import SQLALCHEMY_DATABASE_URL
+from agr_literature_service.api.models import ModModel, ReferencefileModel, ReferenceModel
+from agr_literature_service.api.routers.okta_utils import OktaAccess
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"options": "-c timezone=utc"})
+    new_session = sessionmaker(bind=engine, autoflush=True)
+    db = new_session()
+    mod_abbreviations = [mod.abbreviation for mod in db.query(ModModel.abbreviation).all()]
+    for mod_abbreviation in mod_abbreviations:
+        for ref_id in get_ref_ids_with_workflow_status(db=db, workflow_atp_id="ATP:0000162", mod_abbreviation=mod_abbreviation):
```

Comment:
Not sure if I am over complicating things but if we had 'convert_to_text_job' as the condition with transition_to = 'text_conversion_needed" then we can search for those in a standard way.